src/utils/pause-helper.js

```diff
@@ -39,6 +42,9 @@ export class PauseHelper {
     }
     this.isPlaying_ = isPlaying;
     if (isPlaying) {
+      // Reset hasSize to ensure that the very first measurement doesn't
+      // immediately pause the element.
```

Comment:
I would clarify further: "Pause will not be called until transitioning from 'has size' to 'no size', which means a measurement must first be received that has size, then a measurement that does not have size."

Reply:
+1

Reply:
Done.